### ISSUE TITLE
Is “Spirit Healer Path” still work or not?

### ISSUE THREAD

**Issue by Billdong1997** [Xian55/WowClassicGrindBot#60]
Is “Spirit Healer Path” still work or not? I noticed you have remove all the Spirit Healer Path. I know that the bot use PPather to make a corpse run, but i think the "spirit healer path" should reserve. I have been stuck in this corner in Barrens at least 10 times LOL. I think we should have a very short spirit healer path to make sure that the character run out of the graveyard which is very easy to get stuck in. 
![QQ截图20210519170015](https://user-images.githubusercontent.com/83825966/118787791-b9310e00-b8c5-11eb-9f31-c4b1447c6858.jpg)

**Comment by Billdong1997** [comment]
this is not a bug, i dont know how to cancel the bug label

**Comment by Xian55** [comment]
`Spirit Healer Path` almost rooted out from the code. its not intended to use anymore

**Comment by Leerael** [comment]
TLDR:
- Add 'Joining paths' that force player to follow when conditions are met.
- Add 'Close to Joining Path End' condition to stop the joining path and perform some action(s) (could be walking forward for x seconds, starting another joining path, or auto pathing to the next 'actual path').
..Scenario 1 - (Death Path) Running around something after dying in order to reach corpse
..Scenario 2 - (Zoning Path) Zoning from one zone to another to start next pathing area


The Meat:

Needing something right now where when the player dies they should follow a set path (like the path to vendor), and when they reach 'close to the end' of that path it is considered completed until next death and then tries to path its way back to the corpse like it normally does. This would help me navigate past mountains etc. that keep catching my character in bottoms of crevices and such.

using this same 'joining path' and 'close to joining path end' logic we could essentially set up a between zone path that when it reaches 'close to joining path end' (within a threshold) it then just presses the forward button until the zone is crossed (keep pressing forward and check if zone ID has changed) and then starts the next 'joining path' towards the next 'grinding path' which will start the grind again.

The logic to fire the zoning joining path could be something like 'level=10, zone=elwynn forest' and that could fire the 'elwynn to westfall' joining path, when we hit 'close to the end' for that path we push forward button until zone changes from elwynn to westfall, then we check our last path, if it =elwynn to westfall then we start the 'westfall to next grinding path' joining path.

I imagine we would employ something like current_path, last_path logic to allow us to know which was the last path used.

**Comment by Leerael** [comment]
using this same 'joining path' and 'close to joining path end' logic we could essentially set up a between zone path that when it reaches 'close to joining path end' (within a threshold) it then just presses the forward button until the zone is crossed (keep pressing forward and check if zone ID has changed) and then starts the next 'joining path' towards the next 'griding path' which will start the grind again.

**Comment by Xian55** [comment]
Gooday, 

Please do not reopen ~4 years old issues. It brings no benefit to the conversation.

I really appreciate your post i would be more than happy to engage the conversation, but this is not the place to thinking about your feature request or question.

Continue [here](https://github.com/Xian55/WowClassicGrindBot/issues/new/choose)
